lib/Jsrt/JsrtSourceHolder.h

```diff
@@ -17,6 +17,9 @@ namespace Js
         TUnloadCallback scriptUnloadCallback;
         JsSourceContext sourceContext;
 
+#ifndef NTBUILD
```

Comment:
simplified other `ifdefs`  this one remains to keep unused variable from NTBUILD